### PR TITLE
Additional products

### PR DESCRIPTION
**Pull request by jajh90**

### CODE REVIEW

File: client/dist/styles/app.css [hunk 1]
```diff
@@ -2,7 +2,9 @@
 @import url('product-overview.css');
 @import url('questions-and-answers.css');
 @import url('reviews.css');
-
+@import url('relatedCarousel.css');
+@import url('relatedModal.css');
+@import url('relatedOutfits.css');
```
Comment (thegraceoflayla) [inline client/dist/styles/app.css:7]: Thanks for adding these import statements!

File: client/src/components/App.jsx [hunk 2]
```diff
@@ -168,13 +155,18 @@ class App extends React.Component {
           </div>
         </header>
         <ProductOverview item={this.state.item} reviews={this.state.reviews} />
-        <AdditionalProducts />
+        <AdditionalProducts
+          relatedItems={this.state.relatedItems}
+          fetch={this.fetch}
+          loadPage={this.loadPage}
```
Comment (thegraceoflayla) [inline client/src/components/App.jsx:161]: I love that we're able to share this functions in App.

**Review by ryanlott168 — APPROVED** [review]

File: package.json [hunk 3]
```diff
@@ -36,7 +36,8 @@
     "nodemon": "^2.0.12",
     "path": "^0.12.7",
     "react": "^17.0.2",
-    "react-dom": "^17.0.2"
+    "react-dom": "^17.0.2",
+    "react-icons": "^4.2.0"
```
Comment (thegraceoflayla) [inline package.json:40]: I need to learn more about React icons. I didn't realize they were a thing.

**Review by thegraceoflayla — APPROVED** [review]
These changes look good!

**Pull request merged**